### PR TITLE
Ease validation for `minContext` and `maxContext` (#2601)

### LINKED ISSUE #2601: Update validation copy on "Create WGL opening" Proposal
Update the validation copy of the top field  to ... equal or greater than {{execution constraint value}}

### PR DESCRIPTION
**Pull request by traumschule**
Fixes #2601

### CODE REVIEW

File: packages/ui/src/common/utils/validation.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const maxContext = (
     }
 
     const validationValue = new BN(get(this.options.context, contextPath))
-    if (validationValue && validationValue.lt(new BN(value))) {
+    if (validationValue && validationValue.lte(new BN(value))) {
```
Comment (thesan) [inline packages/ui/src/common/utils/validation.tsx:36]: The previous logic is correct:
It returns an error if `validationValue` is strictly less than `value`.
So `value` is valid when it is greater or equal to `validationValue`

So only the validation messages were wrong

File: packages/ui/src/common/utils/validation.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ export const minContext = (
     }
 
     const validationValue = new BN(get(this.options.context, contextPath))
-    if (validationValue && validationValue.gt(new BN(value))) {
+    if (validationValue && validationValue.gte(new BN(value))) {
```
Comment (thesan) [inline packages/ui/src/common/utils/validation.tsx:66]: Same thing here

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/common/utils/validation.tsx (hunk 1 above)
Comment (traumschule) [reply packages/ui/src/common/utils/validation.tsx:36]: Makes sense.

File: packages/ui/src/common/utils/validation.tsx (hunk 2 above)
Comment (traumschule) [reply packages/ui/src/common/utils/validation.tsx]: Fixed in [46462da](https://github.com/Joystream/pioneer/pull/3793/commits/46462da0b5188b7e92efb660b0ff707266d56083)

**Review by thesan — APPROVED** [review]
Sorry for the delay I forgot about this one. Whenever you can please try to remember to re-request reviews after a PR is fixed

**Pull request merged**